python/nano/notebooks/hpo/custom.ipynb

```diff
@@ -0,0 +1,1609 @@
+      "Epoch 8/24\n",
+      "19/19 [==============================] - 1s 28ms/step - loss: 0.3701 - accuracy: 0.9029 - val_loss: 0.3955 - val_accuracy: 0.8883\n",
+      "Epoch 9/24\n",
+      "19/19 [==============================] - 1s 33ms/step - loss: 0.3393 - accuracy: 0.9133 - val_loss: 0.3831 - val_accuracy: 0.8933\n",
+      "Epoch 10/24\n",
+      "19/19 [==============================] - 1s 30ms/step - loss: 0.3181 - accuracy: 0.9146 - val_loss: 0.3597 - val_accuracy: 0.8950\n",
+      "Epoch 11/24\n",
+      "19/19 [==============================] - 1s 28ms/step - loss: 0.3000 - accuracy: 0.9179 - val_loss: 0.3494 - val_accuracy: 0.8983\n",
+      "Epoch 12/24\n",
+      "19/19 [==============================] - 1s 30ms/step - loss: 0.2842 - accuracy: 0.9217 - val_loss: 0.3541 - val_accuracy: 0.8883\n",
+      "Epoch 13/24\n",
+      "19/19 [==============================] - 1s 40ms/step - loss: 0.2727 - accuracy: 0.9246 - val_loss: 0.3414 - val_accuracy: 0.8967\n",
+      "Epoch 14/24\n",
+      "19/19 [==============================] - 0s 26ms/step - loss: 0.2597 - accuracy: 0.9283 - val_loss: 0.3437 - val_accuracy: 0.8967\n",
+      "Epoch 15/24\n",
+      "19/19 [==============================] - 1s 38ms/step - loss: 0.2509 - accuracy: 0.9308 - val_loss: 0.3526 - val_accuracy: 0.8917\n",
+      "Epoch 16/24\n",
+      "19/19 [==============================] - 1s 30ms/step - loss: 0.2388 - accuracy: 0.9325 - val_loss: 0.3458 - val_accuracy: 0.8983\n",
+      "Epoch 17/24\n",
+      "19/19 [==============================] - 1s 30ms/step - loss: 0.2329 - accuracy: 0.9379 - val_loss: 0.3409 - val_accuracy: 0.9000\n",
+      "Epoch 18/24\n",
+      "19/19 [==============================] - 1s 32ms/step - loss: 0.2236 - accuracy: 0.9388 - val_loss: 0.3291 - val_accuracy: 0.8983\n",
+      "Epoch 19/24\n",
+      "19/19 [==============================] - 1s 40ms/step - loss: 0.2184 - accuracy: 0.9396 - val_loss: 0.3293 - val_accuracy: 0.9000\n",
+      "Epoch 20/24\n",
+      "19/19 [==============================] - 1s 28ms/step - loss: 0.2100 - accuracy: 0.9400 - val_loss: 0.3453 - val_accuracy: 0.8967\n",
+      "Epoch 21/24\n",
+      "19/19 [==============================] - 1s 34ms/step - loss: 0.2048 - accuracy: 0.9454 - val_loss: 0.3218 - val_accuracy: 0.9017\n",
+      "Epoch 22/24\n",
+      "19/19 [==============================] - 1s 35ms/step - loss: 0.1991 - accuracy: 0.9433 - val_loss: 0.3344 - val_accuracy: 0.9000\n",
+      "Epoch 23/24\n",
+      "19/19 [==============================] - 1s 32ms/step - loss: 0.1935 - accuracy: 0.9479 - val_loss: 0.3312 - val_accuracy: 0.9033\n",
+      "Epoch 24/24\n",
+      "19/19 [==============================] - 1s 32ms/step - loss: 0.1875 - accuracy: 0.9504 - val_loss: 0.3425 - val_accuracy: 0.9000\n",
+      "32/32 - 1s - loss: 0.3790 - accuracy: 0.8760 - 832ms/epoch - 26ms/step\n",
+      "Test loss: 0.37904876470565796\n",
+      "Test accuracy: 0.8759999871253967\n"
```

Comment:
This test accuracy is too low for MNIST. We need some tuning on this one. Use val_accuracy for target metric and try again.